.gitignore

```diff
@@ -17,7 +17,7 @@ build
 /src/version.h
 
 # for package files
-lpac-*.deb
+lpac_*.deb
```

Comment:
why change filename pattern?

Reply:
Please see https://github.com/estkme-group/lpac/pull/109#issue-2324688290 and https://github.com/estkme-group/lpac/issues/99#issuecomment-2138031867